autoimpute/imputations/dataframe/series_imputer.py

```diff
@@ -0,0 +1,153 @@
+"""This module performs a series of multiple imputations of missing features
+in a dataset.
+
+This module contains one class - the SeriesImputer. Use this class to
+impute each Series within a DataFrame multiple times using an iteration of fits
+and transformations to reach a stable state of imputation each time. This
+extension of MultipleImputer makes the same imputation methods available as its
+parent class - both univariate and multivatiate. Each method runs `n` times on
+its specified column. When `n` passes through the columns are complete, the
+MultipleImputer returns the `n` imputed datasets. For each of these `n`
+imputations, the method (re)fits and applies imputation to the dataset `k`
+times. Typically `k` should be at least 3 to reach a stable state.
+"""
+
+from autoimpute.utils import check_nan_columns
+from .multiple_imputer import MultipleImputer
+
+
+class SeriesImputer(MultipleImputer):
```

Comment:
Thoughts on naming this the `ChainedImputer` or the `MiceImputer` (or something similar)? I like `MiceImputer`, but open to ideas on naming.

Reasons I'd like to rename this class:
1. `SeriesImputer` has a different meaning in `autoimpute`. [Each imputation method](https://github.com/kearnz/autoimpute/tree/master/autoimpute/imputations/series) is implemented as a `SeriesImputer`, and `SeriesImputers` implement the `ISeriesImputer` ABC. 
2. The Imputer your proposing here makes significant improvements. Therefore, we should feature it, and I will rewrite the documentation to introduce it as the main class that people should use. A name like `MiceImputer` will make sense to new users, especially those coming from the R world.